codebuild/codebuild.config

```diff
@@ -126,5 +126,3 @@ env: TESTS=tls SAW=true GCC_VERSION=NONE
 [CodeBuild:s2nSawHmacFailure]
 snippet: UbuntuBoilerplate2XL
 env: TESTS=sawHMACFailure SAW=true
-
-# Sidetrail
```

Comment:
This file is used to create the codebuild jobs via cloudformation- since the automation isn't done for this, the CodeBuild job was created manually to test #1479  with:

```
# Sidetrail - Run both tests in one job
[CodeBuild:s2nSidetrail]
snippet: UbuntuBoilerplate2XL
env: TESTS=sidetrail
```

Reply:
Ahh I see. I've put that back.